registry/kubevirt-group/kubevirtgroup.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: kubevirtgroup
```

Comment:
kubevirt being the project name and the umbrella name was confusing. So I just made up this name for my sanity.